code/game/area/areas/shuttles.dm

```diff
@@ -10,6 +10,7 @@
 	always_unpowered = FALSE
 	valid_territory = FALSE
 	icon_state = "shuttle"
+	canSmoothWithAreas = /area/shuttle
```

Comment:
in initialize just have it set canSmoothWithAreas to the same type as itself for shuttle areas if the var is empty